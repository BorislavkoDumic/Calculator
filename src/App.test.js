import React from "react";
import CalculatorContainer from "./containers/CalculatorContainers";
import reducer from "./reducers";
import { createStore } from "redux";
import { Provider } from "react-redux";
import renderer from "react-test-renderer";
describe("Snapshot, container renders correctly", () => {
  it("Matches the snapshot", () => {
    const store = createStore(reducer);
    const tree = renderer
      .create(
        <Provider store={store}>
          <CalculatorContainer />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
