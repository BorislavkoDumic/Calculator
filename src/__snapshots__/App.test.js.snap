// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot, container renders correctly Matches the snapshot 1`] = `
<div
  className="App"
>
  <div
    className="calc-wrapper"
  >
    <div>
      <span
        className="input"
      >
        0
      </span>
    </div>
    <div
      className="row"
    >
      <button
        className="button"
        onClick={[Function]}
      >
        7
      </button>
      <button
        className="button"
        onClick={[Function]}
      >
        8
      </button>
      <button
        className="button"
        onClick={[Function]}
      >
        9
      </button>
      <button
        className="button-operator"
        onClick={[Function]}
      >
        +
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="button"
        onClick={[Function]}
      >
        4
      </button>
      <button
        className="button"
        onClick={[Function]}
      >
        5
      </button>
      <button
        className="button"
        onClick={[Function]}
      >
        6
      </button>
      <button
        className="button-operator"
        onClick={[Function]}
      >
        /
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="button"
        onClick={[Function]}
      >
        1
      </button>
      <button
        className="button"
        onClick={[Function]}
      >
        2
      </button>
      <button
        className="button"
        onClick={[Function]}
      >
        3
      </button>
      <button
        className="button-operator"
        onClick={[Function]}
      >
        *
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="button"
        onClick={[Function]}
      >
        0
      </button>
      <button
        className="button"
        onClick={[Function]}
      >
        .
      </button>
      <button
        className="button"
        onClick={[Function]}
      >
        =
      </button>
      <button
        className="button-operator"
        onClick={[Function]}
      >
        -
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="button-operator"
        onClick={[Function]}
      >
        C
      </button>
    </div>
  </div>
</div>
`;
